examples/loki/collect-stdout-with-customzie-parser.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: nginx-app-2
+spec:
+  components:
+    - name: nginx-comp
+      type: webservice
+      properties:
+        image: nginx:1.14.2
+        ports:
+          - port: 80
+            expose: true
+      traits:
+        - type: stdout-logs-collector
+          properties:
+            parser: customize
+            customizeParserVrl: |
```

```suggestion
            VRL: |
```

You've already have customize and parser in the line above, it's redundant information.

---

BTW, the VRL are all uppercase letter accroding vector official site.